File: examples/raw-python/oauth_client_credentials.py
from payxcommerce import DEFAULT_BASE_URL, oauth_client_credentials, print_response

CLIENT_ID = "YOUR_DEVELOPER_APP_CLIENT_ID"
CLIENT_SECRET = "YOUR_DEVELOPER_APP_CLIENT_SECRET"

print_response(oauth_client_credentials(DEFAULT_BASE_URL, CLIENT_ID, CLIENT_SECRET))

File: examples/raw-nodejs/oauth-client-credentials.js
'use strict';

const { DEFAULT_BASE_URL, oauthClientCredentials, printResponse } = require('./payxcommerce');

const clientId = 'YOUR_DEVELOPER_APP_CLIENT_ID';
const clientSecret = 'YOUR_DEVELOPER_APP_CLIENT_SECRET';

oauthClientCredentials(DEFAULT_BASE_URL, clientId, clientSecret).then(printResponse).catch((error) => {
  console.error(error.message);
  process.exit(1);
});
